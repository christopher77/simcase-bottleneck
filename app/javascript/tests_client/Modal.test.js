import React from "react";
import { render } from "@testing-library/react";
import Modal from "../components/ui/Modal";

beforeAll(() => {
  let modalRoot = document.createElement("div");
  modalRoot.setAttribute("id", "modal-root");
  document.body.append(modalRoot);
});

afterAll(() => {
  const elem = document.getElementById("modal-root");
  elem.parentNode.removeChild(elem);
});

describe("<Modal/>", () => {
  it("Modal component snapshot", () => {
    const { baseElement } = render(<Modal />);
    expect(baseElement).toMatchSnapshot();
  });
});
