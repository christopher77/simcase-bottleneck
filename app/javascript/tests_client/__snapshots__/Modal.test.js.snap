// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Modal/> Modal component snapshot 1`] = `
<body>
  <div
    id="modal-root"
  >
    <div
      class="modal"
    />
    <div
      class="card"
    />
  </div>
  <div />
</body>
`;
